### DISCUSSION TITLE
为啥我的绳网界面一直加载中……

### DISCUSSION THREAD

**Discussion by cold-heart-314** [share121/inter-knot#1122] [其他]
![6b5ce225997c094611f263ec7a4bfe8](https://github.com/user-attachments/assets/e95a26c3-e8a5-4a13-ba15-64123bf07c47)
直接在加载。Github正常进，网速应该没问题

**Comment by ijog36** [comment]
希腊奶ଘ(੭ˊᵕˋ)੭* ੈ✩

**Reply by cold-heart-314** [reply]
发帖和看帖在github里全都可以，那个绳网的网页就看不到
